portfolio/src/main/webapp/slideShowScript.js

```diff
@@ -0,0 +1,3 @@
+
+  showSlides();
+  setTimeout(showSlidesAuto, 5000); // Change image every 5 seconds
```

Comment:
Nit: Add a new line at the end of every file so GitHub doesn't add a big red marker in code reviews.

Alternatively, search for "<my code editor> auto add trailing newlines" to set this up once and then never think about it again.